src/alert.html

```diff
@@ -51,6 +51,11 @@ <h1 class="govuk-heading-xl alerts-icon__container alerts-icon__container--48">
   </h1>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds alert-icon__container">
+      {% if alert_data.is_expired %}
+      <h2 class="govuk-heading-s">
+        Ended <time datetime="{{ alert_data.expires_date.as_iso8601 }}">{{ alert_data.expires_date.as_lang }}</time>
```

Comment:
Is "ended" accurate here? For example, if the alert went:

```
Chemical leak in XYZ

Ended at ...
```

I'd read that as "the chemical leak has stopped", which may not be true. How about something more accurate, like "Broadcast ended at ...".

We use the term "broadcast" on the home page, so it seems reasonable to reuse it here.

Reply:
Yeah, I think you’re right. I’ve changed it to use the ‘Stopped sending’ version from Karl’s doc. This makes the language consistent because we say ‘Sent at’ beneath the alert text. I’ve amended the commit and added a screenshot to the description.

Reply:
To make it even more consistent, should we say 'Started sending' instead of 'Sent at' because it is continually sent at 60 second intervals? And someone might receive the alert 3 hours after it begins

Reply:
Thanks @quis. I think you missed an `at` in the code (comparing with the screenshot).

Reply:
The `at` is in the model:
https://github.com/alphagov/notifications-govuk-alerts/blob/eb35989cfc29971608304ba538bbf59f33797938/lib/alert_date.py#L25

Reply:
Ahh, I see, thanks for explaining. I think there's some merit in @idavidmcdonald's suggestion too, but this change is also good on its own.